packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.stories.ts

```diff
@@ -1,17 +1,17 @@
 import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
 import type { Meta, StoryObj } from "@storybook/vue3";
 import { Default as ListboxDefaultStory } from "../OnyxListbox/OnyxListbox.stories";
-import OnyxSelect from "./OnyxSelect.vue";
+import OnyxSelectInput from "./OnyxSelectInput.vue";
 
-// TODO: description in figma is missing.
 /**
- * This is a select.
+ * This is the select element.
+ * Itself is readonly and only takes care of showing the selection.
  */
-const meta: Meta<typeof OnyxSelect> = {
-  title: "components/Select",
+const meta: Meta<typeof OnyxSelectInput> = {
+  title: "support/SelectInput",
```

Comment:
Could you add a changeset as breaking change that the component was renamed?

Reply:
done